zulipterminal/config/ui_mappings.py

```diff
@@ -40,3 +40,10 @@
     400: {"bool": "", "name": "Member"},
     600: {"bool": "is_guest", "name": "Guest"},
 }
+
+STREAM_POST_POLICY = {
```

Comment:
This is not the same as in the title of the commit?

Reply:
This was added under the first commit which made changes only to `ui_mappings.py` so that it could be merged using `git cherry-pick` if required.